app/components/common/Header/NavBar/index.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+import theme from '@/styles/theme';
+import { usePathname, useRouter } from 'next/navigation';
+import styled from 'styled-components';
+
+interface NavDataProps {
+  title?: string;
+  path?: string;
+}
+interface NavProps {
+  type?: 'top' | 'main';
+  data?: NavDataProps[];
+}
+const MultiNavBar = ({ type = 'top', data }: NavProps) => {
+  const router = useRouter();
+  const path = usePathname();
+  return (
+    <NavWrapper type={type}>
+      {data &&
+        data.map((navData, idx) => (
+          <ButtonWrapper
+            onClick={() => router.push('/' + navData.path)}
+            key={idx}
+            type={type}
+            active={path === '/' + navData.path ? 'true' : 'false'}
+          >
+            {navData.title}
+          </ButtonWrapper>
+        ))}
+      {type === 'top' ? ( // 알람과 modal창을 띄워야해서 따로 분리 했습니다.
+        <>
+          <ButtonWrapper
+            type={type}
+            onClick={() => console.log('알람 이벤트 처리')}
+          >
+            알림
+          </ButtonWrapper>
+          <ButtonWrapper type={type} onClick={() => router.push('/login')}>
+            로그인
+          </ButtonWrapper>
+        </>
+      ) : (
+        ''
+      )}
+    </NavWrapper>
+  );
+};
+const NavWrapper = styled.nav<{ type: string }>`
+  height: ${(props) => (props.type === 'top' ? '15px' : '42.5px')};
+  font-size: ${(props) =>
+    props.type === 'top' ? theme.fontSize.caption3 : theme.fontSize.title2};
+  position: ${(props) => (props.type === 'top' ? 'absolute' : 'static')};
+  top: 0;
+  right: 0;
+`;
+const ButtonWrapper = styled.button<{ type: string; active?: string }>`
+  width: ${(props) => (props.type === 'top' ? '70px' : '100px')};
```

Comment:
```suggestion
  width: ${(props) => (props.type === 'top' ? '7rem' : '10rem')};
```
아래의 코드들도 rem으로 수정해주세요..!

Reply:
넵 알겠습니다!